docs/conf.py

```diff
@@ -175,3 +175,4 @@
      }
 
 subprocess.run(["make fledge_plugins.rst"], shell=True, check=True)
```

Comment:
We can call multiple targets like `"make fledge_plugins.rst plugins"`
Also can we delete `fledge_plugins.rst` checked-in file from GitHub? As now we have option to build it from conf.py?

Reply:
Created a composite target to make generated documentation